concordia/views.py

```diff
@@ -795,6 +804,10 @@ def add_transcription_status_summary_to_projects(self, projects):
             project.transcription_statuses
 
 
+reserve_rate = lambda g, r: None if r.user.is_authenticated else "1/m"
```

Comment:
This might want to be a bit higher since we trigger requests at 1/m and timing skew could hit it. I'm thinking 2-3/m should suffice for a single page but how much do we care about people having multiple windows open?